### DISCUSSION TITLE
希望添加 词库回复功能

### DISCUSSION THREAD

**Discussion by CharlotteLFXY** [FloatTech/ZeroBot-Plugin#234] [Ideas]
功能1.添加词条   指令名+空格+关键词+空格+ 回复内容 （如“添加关键词 一加一等于几 二”）
功能2.删除词条   指令名+空格+关键词   （如“删除关键词 一加一等于几”）
功能3.匹配模式   指令名+空格+关键词+匹配模式 （如“匹配模式 一加一等于几 模糊”，则含有一加一等于几的消息都触发词条）

功能1拓展：指令名+空格+关键词+空格+ 回复内容 +空格+ 回复内容+空格+ 回复内容+... （同一关键词随机输出不同回复）

功能拓展：回复内容如果能适配表情甚至图片，那将绝杀
功能拓展：定时回复功能，虽然插件里已经有定时指令触发器但是配置起来不太方便，
指令名+空格+时间+空格+内容
（如“添加定时提醒 *-15-30-00  中午三点半了该摸鱼了！”）每天15：30定时回复
（如“添加定时提醒 7-20-30-00  明天上班了！”）每周日20：30定时回复

**Comment by CharlotteLFXY** [comment]
可参考 https://www.wolai.com/fcraft/jnRhGsLMAqoSTg7oFJSebp

**Comment by MoYoez** [comment]
https://github.com/FloatTech/ZeroBot-Plugin/commit/c4cbb0cd590e6f257e97ac106dfa7a45731b9a9b


这个实现了

**Reply by fumiama** [reply]
> [c4cbb0c](https://github.com/FloatTech/ZeroBot-Plugin/commit/c4cbb0cd590e6f257e97ac106dfa7a45731b9a9b)
> 
> 这个实现了

是的

**Reply by CharlotteLFXY** [reply]
[我|大家|有人][说|问][正则表达式]你[答|说|做|执行][模版]

能否举几个例子讲解下正则表达式和模版具体是什么形式？

**Reply by CharlotteLFXY** [reply]
略微测试了下，好像有个BUG，正则表达式里有QQ表情好像无法正确回复，模版里有表情可以回复

**Reply by CharlotteLFXY** [reply]
另：“查看所有触发指令”好像无法使用
